sdk/servicebus/azure-messaging-servicebus/src/test/java/com/azure/messaging/servicebus/ServiceBusReceiverAsyncClientIntegrationTest.java

```diff
@@ -64,12 +63,6 @@
     private ServiceBusReceiverAsyncClient receiver;
     private ServiceBusSenderAsyncClient sender;
 
```

Comment:
There is a `messagesDeferredPending` that we no longer use and can be moved or removed. We never read its values.